go/private/actions/link.bzl

```diff
@@ -98,6 +103,11 @@ def emit_link(
     if go.mode.link == LINKMODE_PLUGIN:
         tool_args.add("-pluginpath", archive.data.importpath)
 
+    # TODO: Remove when https://github.com/bazelbuild/bazel/pull/12304 is mainstream
```

Comment:
Do you know what version this will be? 5.0.0 it sounds like?

Reply:
Apparently. When that's widespread enough, it seems we'll be able to get the so name in starlark. It's not very clear to me yet how though.